src/teensy_sketch/teensy_sketch.ino

```diff
@@ -96,5 +142,137 @@ void runMainLoop() {
 
 void runInitLoop() {
   // TODO: ESP handshaking and blocking to wait on commands
+  while (1);
 }
 
+// ------------------------------------------------------------
+//                   Function definitions
+// ------------------------------------------------------------
+
+// REGION DATA COLLECTION FUNCTIONS
+float getAltitude(){
```

Comment:
This function has a whole lot of magic numbers, not sure if it would be nicer to define them as constants or as variables at the start of the function.